src/main/java/graphql/schema/DataFetchingEnvironment.java

```diff
@@ -44,11 +45,9 @@
      * Returns a context argument that is set up when the {@link graphql.GraphQL#execute(String, Object)} method
      * is invoked
      *
-     * @param <T> you decide what type it is
-     *
      * @return a context object
      */
-    <T> T getContext();
+    ExecutionContext getContext();
```

Comment:
So this is an outright API breaking change.  The previous calls expected to get the original "context object"

So from a SemVer point of view we cant make this change inside a minor version.

Even if we made a Major SemVer change I wont not do it this way.  If we wanted to expose ExecutionContext then we would just add it as another attribute and hence its a non breaking change